docs/UserGuide.md

```diff
@@ -140,18 +140,19 @@ Format: `edit INDEX [/nNAME] [/pPHONE] [/eEMAIL] [/aADDRESS] [/tTAG]…​`
 * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
 * You can remove all the person’s tags by typing `/t` without
   specifying any tags after it.
+* Role is not allowed to be edited!
```

Comment:
Small nit. Grammar seems abit off here. Perhaps could say "Role is not editable" or "Role cannot be edited".

Reply:
fixed